services/filesstore/s3store.go

```diff
@@ -41,6 +41,8 @@ const (
 	bucketNotFound = "NoSuchBucket"
 )
 
+var ErrNoS3Bucket = errors.New(bucketNotFound)
```

Comment:
The constant string was an error code. We usually have sentences for error strings - "bucket not found"